pytest_plugins/fixture_markers.py

```diff
@@ -57,28 +59,54 @@ def pytest_generate_tests(metafunc):
                         if re.fullmatch(str(matcher[0]), str(setting_rhel_ver))
                     ]
                 )
-        network_params = ['ipv6' if settings.server.is_ipv6 else 'ipv4']
         rhel_params = []
         ids = []
         filtered_versions = set(list_params + match_params)
         # default to all supported versions if no filters were found
         for ver in filtered_versions or settings.supportability.content_hosts.rhel.versions:
             rhel_params.append(dict(rhel_version=ver, no_containers=no_containers))
 
+        # Determine the default network type based on settings
+        if settings.content_host.network_type == 'dualstack':
```

Comment:
Should this be using a constant instead of hardcoding? It's repeated in this file.

Reply:
I will unhardcode it in the whole function.